packages/central-server/src/apiV2/userFavouriteDashboardItem/GETUserFavouriteDashboardItems.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { GETHandler } from '../GETHandler';
+
+export class GETUserFavouriteDashboardItems extends GETHandler {
+  async assertUserHasAccess() {
+    this.req.flagPermissionsChecked(); // bypass permissions assertion
+  }
+
+  async findRecords(criteria, options) {
+    const { user_id: userId } = JSON.parse(this.req.query.filter);
+    const loggedUserId = this.req.userId;
+    if (!userId || userId !== loggedUserId) {
+      return [];
+    }
+
+    return this.database.find(this.recordType, criteria, options);
+  }
```

Comment:
```suggestion
  async findRecords(criteria, options) {
    const loggedUserId = this.req.userId;
    if (!loggedUserId) {
      return [];
    }

    return this.database.find(this.recordType, { ...criteria, user_id: loggedUserId }, options);
  }
```

Seems like this is only going to return the `userFavouriteDashboardItems` of the logged in user anyway right? I like that, makes sense. So then you could change this to not need a `user_id` query parameter at all 👍